include/mbedtls/config_psa.h

```diff
@@ -351,8 +352,11 @@ extern "C" {
 #endif /* PSA_WANT_ALG_XTS */
 
 #if defined(PSA_WANT_ALG_ECB_NO_PADDING)
+#if !defined(MBEDTLS_PSA_ACCEL_ALG_ECB_NO_PADDING) || \
+    defined(PSA_HAVE_SOFT_BLOCK_CIPHER)
 #define MBEDTLS_PSA_BUILTIN_ALG_ECB_NO_PADDING 1
 #endif
+#endif /* PSA_WANT_ALG_ECB_NO_PADDING */
 
```

Comment:
`MBEDTLS_PSA_ACCEL_ALG_STREAM_CIPHER` is missing as well. The key type would be important as well, but `MBEDTLS_PSA_ACCEL_ALG_STREAM_CIPHER` has to be there for the dispatch code, doesn't it?

Reply:
Updated (with more fixes related to the interaction between key type ChaCha20 and algorithms stream cipher and chacha20-poly1305